syncmaster/worker/transfer.py

```diff
@@ -20,14 +22,15 @@
 from syncmaster.worker.settings import WorkerAppSettings
 
 logger = get_task_logger(__name__)
+load_correlation_ids()
+load_celery_current_and_parent_ids()
```

Comment:
Shouldn't we call these in after_setup_task_logger or before_setup_task_logger?

Reply:
no, we should call it globally when starting celery app, then any other hooks will be able to retrieve correlation id from headers and set it in celery tasks

Reply:
I see, this call actually creates its own hooks:
https://github.com/snok/asgi-correlation-id/blob/ed006dcc119447bf68a170bd1557f6015427213d/asgi_correlation_id/extensions/celery.py#L14

Do we actually need `transfer_correlation_id` and `load_correlation_id` then?

Reply:
we do not need, thereby we could also remove extra worker setting: `CORRELATION_CELERY_HEADER_ID `